stories/core/Surface.stories.tsx

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import { CreeveyMeta } from 'creevey';
+import React from 'react';
+import { Surface, SurfaceProps } from '../../src/core';
+
+export default {
+  component: Surface,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
```

Comment:
remove `id` too

Reply:
btw, I dont remember, what we have decided and why want to show prop, but disable control.. Is it just so people are aware of these props are existing?
Cause otherwise we could fully hide the control of those and not bother `{ table: { disable: true } }`...

Reply:
Removed `id`